src/data/rds/2014-10-31/api-2.json

```diff
@@ -1844,6 +1844,7 @@
         "TargetDBClusterSnapshotIdentifier":{"shape":"String"},
         "KmsKeyId":{"shape":"String"},
         "PreSignedUrl":{"shape":"String"},
+        "DestinationRegion":{"shape":"String"},
```

Comment:
Does the model need to be modified if the member is being added in `Aws\Rds\RdsClient::applyDocFilters`?

Reply:
Yes, `applyDocFilters` functions are only called at time of doc building, not at time of json model compiling.